release-notes/2023-10/README.md

```diff
@@ -0,0 +1,252 @@
+                - `dataframeservice.ingestion.kafkaBackend.enabled`
+                - `dataframeservice.kafkacluster.kafka.enabled`
+                - `dataframeservice.schema-registry.enabled`
+            - Run a Helm upgrade
+            - Wait for the `dfs-kafka` pods to disappear from the cluster
+        4. Remove the Strimzi Kafka Operator from the cluster
+            - Set the following Helm value to false in the `systemlink-admin-values.yaml` file:
+                - `strimzi-kafka-operator.enabled`
+            - Run a Helm upgrade
+        5. Remove the CRDs for the Strimzi Kafka Operator from the cluster. By design, these are not removed when the operator is uninstalled, so they need to be cleaned up manually. Run `kubectl delete -f systemlinkadmin/charts/strimzi-kafka-operator/crds` to delete the CRDs.
+        6. Delete the Persistent Volume Claims (PVCs) for the Kafka-related pods. Looks for PVCs containing "dfs-kafka" in Lens.
+    - After completing these steps, if you need to update SystemLink Enterprise again, you should skip steps 2 and 3 of the updating instructions for updating the Strimzi Kafka Operator CRDs, to avoid recreating the unneeded CDs.
+- dataframeservice 0.14.39
+    - Default memory request and limit increased from 2GB per DataFrame Service pod to 4GB. Disabling Kafka (see separate instructions) will greatly reduce overall resource usage for the cluster.
+    - Since data tables created after updating won't consume additional Kafka resources, you can likely decrease the dataframeservice.kafkaconnect.spec.resources.requests.memory and/or dataframeservice.kafkacluster.kafka.resources.requests.memory values if required to fit the larger DataFrame Service pods until Kafka is disabled.
+    - Note that appendable tables created prior to the update will continue to consume Kafka resources.
+- The Test Analytics privilege category has been added, and includes the Query Measurements privilege. This privilege is not currently functional and is being added in support of features that will release in a future version.
+- testinsightsui 0.6.107
+    - Comments service was added to top level helm chart in previous release 2023-09. In this release, feature flag is removed and comments tab will be visible in result details page and user can add/view/edit/delete comments under a test result.
+
+## Helm Chart Breaking Changes
+
+- New - mongodb connection string global value override.
+    - Customers can specify the user/password directly in the global ("mongodb+srv://user:pass@host/<`database`>") (<> will be replaced during per-service Helm install/upgrade)
+    - Customers can use per-service user/password combinations ("mongodb+srv://<`username`>:<`password`>@host/<`database`>") (<> will be replaced during per-service Helm install/upgrade)
+- CRITICAL - The systemlink Helm chart defaults to connect to an external MongoDB instance
```

Comment:
This should be the first item in this this.

Reply:
@prestwick  , I did move the Critical section to be the first.

Question: I expect the MongoDB helm chart breaking changes need to be applied by GM regardless of the remaining issues we're still troubleshooting during October:
- See this input from Max Brown [Teams link](https://teams.microsoft.com/l/message/19:26ceeb9c81db41fd9f4e2e36c6cfd787@thread.skype/1697460321784?tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e&groupId=41626d4a-3f1f-49e2-abdc-f590be4a329d&parentMessageId=1696009681604&teamName=ASW%20SystemLink&channelName=General&createdTime=1697460321784)
- `single Mongo validation can wait a bit since R&D is still working out some issues in our own clusters`

Reply:
As I mentioned in feedback on [this PR](https://github.com/ni/install-systemlink-enterprise/pull/169) we did **not** end up changing the default for the October release. This is no longer critical and GM should not have to make changes related to MongoDB for this release.

Reply:
Why is the wording different than what I see in the [`sle-release-notes-2023-10` branch](https://github.com/ni/install-systemlink-enterprise/tree/sle-release-notes-2023-10/release-notes/2023-10#helm-chart-breaking-changes)?

The wording changes in that branch need to be here as well.

Reply:
Hi @g-m-o-n-e-y , the initial info came from this spreadsheet, as documented per our squads:

[SLE 2023-10_Helm Chart Breaking Changes, Software Behavior Changes (including Priviliges) for the Release Notes - out of September 2023.xlsx](https://nio365.sharepoint.com/:x:/s/ASWSystemLink/EcxeSe4JIQdNqp7FP_x9-AQBh2DR8PGflWDWux43xUPsYA?e=NR8lbz)

This comment shows outdated and if I check the Readme.md in this same branch the content is as you have mentioned above:
- https://github.com/ni/install-systemlink-enterprise/blob/sle-release-notes-2023-10/release-notes/2023-10/README.md

Can we then consider this comment as resolved?

FYI @priya-ni